### PR TITLE
Move donate link to header and add GitHub link

### PR DESCRIPTION
**Pull request by pravdomil**
**changelog**
- **fix** donate link is too close to search field
- **add** link to GitHub repo

**before**
donate

**after**
33867102-94ba11e6-defa-11e7-9f48-6b593cad1d7e

### CODE REVIEW

File: frontend/components/header.js [hunk 1]
```diff
@@ -14,11 +14,20 @@ export default () => (
       Hackable &nbsp;
       No tracking
     </p>
+    
+    <p>
+      <a className="link" href="https://github.com/pravdomil/shields">GitHub</a> &nbsp;
```
Comment (paulmelnikow) [inline frontend/components/header.js]: Could you fix this link? ^^

**Comment by pravdomil** [comment]
*after*
after

**Comment by pravdomil** [comment]
please merge or close

**Comment by platan** [comment]
Can we merge it? PR description contains the "after" screenshot.

**Comment by paulmelnikow** [comment]
Hey, sorry for the delay. These links still look confusing to me. It's not obvious they are clickable.

How about using this for GitHub?


https://img.shields.io/github/stars/badges/shields.svg?style=social

And this for OpenCollective?

```
<a href="https://opencollective.com/shields/donate" target="_blank">
  <img src="https://opencollective.com/shields/donate/button@2x.png?color=white" width=200 />
</a>
```
<a href="https://opencollective.com/shields/donate" target="_blank">
  
</a>

**Comment by pravdomil** [comment]
@paulmelnikow sorry I have no future plan to maintain this pull request and I think that github link is still better then no link, please close or merge

**Pull request closed without merging**